xml/deployment_hwrecommend.xml

```diff
@@ -167,6 +168,7 @@
      </step>
     </procedure>
    </sect3>
+<!-- 2020-09-07 tbazant (ceph.conf): How do we set this with ceph-salt? -->
```

Comment:
now, you only should need to care about `public_network`. `mon_host` is managed by cephadm now.